tools/ci_build/github/linux/docker/scripts/training/ortmodule/stage2/requirements.txt

```diff
@@ -0,0 +1 @@
+deepspeed
```

Comment:
what's is stage1 vs. stage2?

Reply:
This is for ORTModule testing and will go away when we migrate to pytorch/ort